src/distributions/mod.rs

```diff
@@ -81,7 +81,6 @@
 //! - Related to real-valued quantities that grow linearly
 //!   (e.g. errors, offsets):
 //!   - [`Normal`] distribution, and [`StandardNormal`] as a primitive
-//!   - [`Cauchy`] distribution
```

Comment:
@rohitjoshi did you intend to move the reference to Cauchy in this documentation? Because you haven't added it back.